### PR TITLE
Ldn 8 / saman zahedi

### PR DESCRIPTION
**Pull request by SamanZahedi**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?
@beckyf0keefe please review my assignment,

### CODE REVIEW

File: css/store.css [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+  cursor: pointer;
+}
+
+/* page two */
+/* Get today section */
+
+main {
+  display: flex;
+  width: 1440px;
+  height: 820px;
```
Comment (beckyf0keefe) [inline css/store.css:59]: Try not to set the height and width of the main section, while 1440px is a standard width for a screen there will be lots of people who are using different sizes, so try not to restrict it, if you want to set the size of the screen, try using vh/vw or %

File: css/store.css [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+.nav-link:hover {
+  color: #e0633a;
+  cursor: pointer;
+}
+
+/* page two */
+/* Get today section */
+
+main {
+  display: flex;
```
Comment (beckyf0keefe) [inline css/store.css:57]: Good use of flex to align the form and the pic next to each other

File: css/store.css [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+  margin-top: 12px;
+}
+
+legend {
+  margin-top: 3.5rem;
+}
+input[type="radio"] + label {
+  display: inline-block;
+}
+
```
Comment (beckyf0keefe) [inline css/store.css:143]: Good understand of combinators

File: css/store.css [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+.details p {
+  color: #e0633a;
+  font-size: 2.3rem;
+  font-weight: 500;
+  margin-top: 0px;
+  margin-bottom: 15px;
+}
+.required::after {
+  content: " *";
+}
```
Comment (beckyf0keefe) [inline css/store.css:85]: Good way to do this!

**Review by beckyf0keefe — COMMENTED** [review]
Looking really good overall, just make sure that you use the form tag so that all your inputs can be grouped and then the special form input button so that the form is checked for all the required fields

File: store.html [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+            <a class="nav-link" href="#">Help</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Login</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="form-section">
```
[the remark this answers is not in the document]
Comment (SamanZahedi) [reply store.html]: Where's my <form> tag I wonder, added thanks

File: store.html [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+              <label for="space-grey">Space Grey</label>
+            </div>
+            <div class="t-cs">
+              <input type="checkbox" id="checkbox1" checked />
+              <label for="checkbox1"
+                >By placing your order you agree to Karma's
+                <a href="#">Terms and Conditions</a>. *
+              </label>
+            </div>
+            <button>Place my order</button>
```
[the remark this answers is not in the document]
Comment (SamanZahedi) [reply store.html]: Changed it to input type="submit" and defined css for it, is this a right way to do it?

File: store.html [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+                <input id="postcode" type="text" required />
+              </div>
+            </div>
+            <div class="checks">
+              <legend>Select a colour</legend>
+              <input id="karma-orange" type="radio" name="color" checked />
+              <label for="karma-orange">Karma orange</label>
+              <input id="space-grey" type="radio" name="color" />
+              <label for="space-grey">Space Grey</label>
+            </div>
```
[the remark this answers is not in the document]
Comment (SamanZahedi) [reply store.html]: value attribute added

File: css/store.css (hunk 1 above)
Comment (SamanZahedi) [reply css/store.css]: Figma bad influence, it's changed to % now

**Comment by SamanZahedi** [comment]
Hi @MatildaAko could you please review my page?

**Review by MatildaAko — COMMENTED** [review]
Very good work, you have a good understanding of most of the concepts of HTML and CSS. Keep it up

**Review by Jacint-M-David — APPROVED** [review]
Very well organised and readable code Saman!

**Pull request closed without merging**